content/neuvector-prime/azure/contents.lr

```diff
@@ -0,0 +1,464 @@
+2. Select existing Resource group from the drop-down list.
+3. Select existing AKS Cluster Name from the drop-down list.
+4. Choose a name for Cluster Extension Resource name as suggested by the
+   tooltip or the following guidance; "Allows alphanumeric, dots, and the value must be between 1-253 characters long".
+
+
+   !["Existng Resource Group"](./basicclusternv.png)
+
+
+
+
+   ---
+   **NOTE**
+
+   The **Create new** Resource group feature is not supported.
+
+   ---
+
+
+
+   C["New Resource Group"](./noresourcegroupver.png)
+
+
+
+   Click "Next".
+
+
+
+## NeuVector configuration
+
+
+1. Choose an Admin password following the criteria in the tooltip.
+
+
+!["Application Details"](./adminpassword.png)
+
+
+Click "Next".
+
+## Review + create
+
+
+  There will be a summary of the offer and a link to "view automation template" (Azure Resource Manager Template)
+
+
+     Price
+     Basics
+     NeuVector Configuration
+
+Click "Create"
+
+## Deployment complete
+
+Deployment will be in progress. After it is completed, the following resources are created
+
+1. AKS Cluster
+2. Kubernetes Service Extension
+
+Click 'Go to Resource group', then click "AKS cluster" (created by offer).
+
+Click 'Connect' and follow the Cloud Shell tab instructions to set the cluster context.
+  1. Set the cluster subscription.
+  2. Download cluster credentials.
+
+
+## Install scanner updater
+
+The NeuVector scanner image is updated daily with a new CVE database on the NeuVector docker hub registry. It is required to install the scanner updater AFTER a successful initial NeuVector deployment.
+
+Download [scanner.yaml](./scanner.yaml).
+Execute the following command in a Cloud Shell or Node which has access to kubectl.
+
+Install scanner:
+```
+kubectl apply  -f scanner.yaml
+```
+The output should be similar to the example below:
+
+```
+deployment.apps/neuvector-scanner-pod created
+cronjob.batch/neuvector-updater-pod created
+```
+
+Check the scanner pods are running:
+```
+kubectl get pods -n neuvector
+```
+
+The output should be similar to the example below:
+```
+NAME                                       READY   STATUS      RESTARTS   AGE
+neuvector-controller-pod-5687c4f89-5cdb6   1/1     Running     0          21h
+neuvector-controller-pod-5687c4f89-dqqmh   1/1     Running     0          21h
+neuvector-controller-pod-5687c4f89-jgnp9   1/1     Running     0          21h
+neuvector-csp-pod-6c6d945488-46tnj         1/1     Running     0          21h
+neuvector-enforcer-pod-28p52               1/1     Running     0          21h
+neuvector-manager-pod-5c75bd44bb-r867s     1/1     Running     0          21h
+neuvector-scanner-pod-9489c5d78-mzthg      1/1     Running     0          7s
+neuvector-scanner-pod-9489c5d78-rzjcz      1/1     Running     0          9s
+neuvector-scanner-pod-9489c5d78-v5xrq      1/1     Running     0          10s
+neuvector-updater-pod-28294240-lpshs       0/1     Completed   0          10m
```

Comment:
The updater entries only show up after the cronjob has been run.  Depending on when they check this they may not see it right away.  You may need to say something like the "Updater cronjob, by default runs once a day. After it has run, you will see a "completed" neuvector-updater-pod-XXXXX-XXXX entry when you check the pods.